docs/docs/guides/generator.mdx

```diff
@@ -13,8 +13,8 @@ The goal of this guide is to develop a Generator and add it to Infrahub, we will
 1. Identify the relevant data you want to extract from the database using a [GraphQL query](/topics/graphql), that can take an input parameter to filter the data
 2. Write a Python script that uses the GraphQL query to read information from the system and generates new data based on the response
 3. Create an entry for the generator within an .infrahub.yml file.
-4. Create a Git repository
-5. Test the generator with infrahubctl
+4. Test the generator with infrahubctl
```

Comment:
Why do we change this order? I actually think it makes sense that we do create a git repository early in the process to teach people that they should use version control during the development cycle.

Reply:
I felt it was weird that we'd write the query and code, but not test it, and upload it to git first. I figured it'd make sense to test before pushing up and then also provide more on how these run when synced and using a PC.

Maybe we can put it at the beginning of the guide to create the repo?

Reply:
> Maybe we can put it at the beginning of the guide to create the repo?
It would make sense to me, this way we keep the version control idea early on as Wim proposed.